core/runtime/module.go

```diff
@@ -0,0 +1,45 @@
+package runtime
+
+import (
+	"fmt"
+	"sync"
+
+	"flamingo.me/dingo"
+	"flamingo.me/flamingo/v3/framework/flamingo"
+	"go.uber.org/automaxprocs/maxprocs"
+)
+
+type (
+	// Module for core.runtime
+	Module struct {
+		logger flamingo.Logger
+	}
+)
+
+var once = sync.Once{}
+
+// Inject Module dependencies
+func (m *Module) Inject(logger flamingo.Logger) *Module {
+	m.logger = logger
```

Comment:
Really just detail and I would change it myself if I would have access to the branch ;-)

What we often do is initialize the logger here:
m.logger = logger.WithField(flamingo.LogKeyModule,"core.runtime").WithField(flamingo. LogKeyCategory,"module")

I also suggest to use a different "category" value or don't use a category for now since its the only log in the module for now

Reply:
makes totally sense ;)